.travis.yml

```diff
@@ -65,19 +60,42 @@ jobs:
         # case that the travis log doesn't produce any output for more than 10 minutes
         - while sleep 540; do echo "==== Still running (travis, don't kill me) ===="; done &
       script:
-        - docker run -e CI -e TRAVIS_JOB_ID="$TRAVIS_JOB_ID" -e TRAVIS_BRANCH="$TRAVIS_BRANCH" p4a /bin/sh -c "$COMMAND"
+        - >
+          docker run
+          -e CI
+          -e TRAVIS_JOB_ID="$TRAVIS_JOB_ID"
+          -e TRAVIS_BRANCH="$TRAVIS_BRANCH"
+          -e ANDROID_SDK_HOME="/home/user/.android/android-sdk"
+          -e ANDROID_NDK_HOME="/home/user/.android/android-ndk"
+          p4a /bin/sh -c "$COMMAND"
       after_script:
         # kill the background process started before run docker
         - kill %1
-      name: Python 3 armeabi-v7a
+      name: Python 3 arm64-v8a
       # overrides requirements to skip `peewee` pure python module, see:
       # https://github.com/kivy/python-for-android/issues/1263#issuecomment-390421054
-      env: COMMAND='. venv/bin/activate && cd testapps/ && python setup_testapp_python3_sqlite_openssl.py apk --sdk-dir $ANDROID_SDK_HOME --ndk-dir $ANDROID_NDK_HOME --requirements libffi,sdl2,pyjnius,kivy,python3,openssl,requests,sqlite3,setuptools' --arch=armeabi-v7a
+      env:
+        COMMAND='. venv/bin/activate && cd testapps/ && python setup_testapp_python3_sqlite_openssl.py apk --sdk-dir $ANDROID_SDK_HOME --ndk-dir $ANDROID_NDK_HOME --requirements libffi,sdl2,pyjnius,kivy,python3,openssl,requests,sqlite3,setuptools' --arch=arm64-v8a
     - <<: *testing
-      name: Python 3 arm64-v8a
-      env: COMMAND='. venv/bin/activate && cd testapps/ && python setup_testapp_python3_sqlite_openssl.py apk --sdk-dir $ANDROID_SDK_HOME --ndk-dir $ANDROID_NDK_HOME --requirements libffi,sdl2,pyjnius,kivy,python3,openssl,requests,sqlite3,setuptools' --arch=arm64-v8a
+      name: Python 3 armeabi-v7a
+      os: osx
+      osx_image: xcode11  # since xcode1.3, python3 is the default interpreter
+      before_script:
+        # installs java 1.8, android's SDK/NDK and p4a
+        - make -f ci/makefiles/osx.mk
+        - export JAVA_HOME=/Library/Java/JavaVirtualMachines/adoptopenjdk-8.jdk/Contents/Home
+        # Run a background process to make sure that travis will not kill our tests in
+        # case that the travis log doesn't produce any output for more than 10 minutes
+        - while sleep 540; do echo "==== Still running (travis, don't kill me) ===="; done &
```

Comment:
That might not be needed for the OSX build actually, because I don't think it would run more than 20 or 30 minutes without log printing.
`travis_wait` waits 20 minutes by default and can be increased up to 30 minutes by passing time as parameter.

Reply:
I maintained the background process...I think that could be useful..., but I removed one line of the above comment (since we already commented before)...but we can remove it if you feel this is useless...just let me know :wink: